### PR TITLE
Move THANKS.txt scipy file to scipy.org website

### PR DESCRIPTION
**Pull request by rlucas7**
On the Scipy-dev list we discussed moving the THANKS.txt file from the scipy repo proper to the website. 
This move will ameliorate the merge conflicts that cause both contributors and maintainers difficulty with many merge conflicts when rebasing off master.

For list dicsussion:
https://mail.python.org/pipermail/scipy-dev/2020-August/024331.html 

I'll also open a corresponding PR in scipy repo to ***remove*** the `THANKS.txt` file from that repository.

### CODE REVIEW

File: www/scipylib/donations.rst [hunk 1]
```diff
@@ -57,6 +57,223 @@ infrastructure:
 *This list is ordered by time (most recent contributions first) and was last
 updated in April 2019.*
 
+
+Thanks
+------
+
+SciPy is a community project sponsored by Enthought, Inc.
```
Comment (rgommers) [inline www/scipylib/donations.rst:64]: That's not quite true anymore, and hasn't been for over a decade .....

File: www/scipylib/donations.rst [hunk 2]
```diff
@@ -57,6 +57,223 @@ infrastructure:
...
+Peterson, and Eric Jones.  Travis Oliphant and Eric Jones each contributed
+about half the initial code.  Pearu Peterson developed f2py, which is the
+integral to wrapping the many Fortran libraries used in SciPy.
+
+Below is a partial list of contributors to SciPy.  If you've
+been left off, please email the `SciPy Developers List`_.
+
+The SciPy development team would like to thank the following individuals for
+submitting issues, bugfixes, enhancements, and development infrastructure:
+
```
Comment (rgommers) [inline www/scipylib/donations.rst:75]: How about we merge the `authors.py` feature first, and then we just make this a complete list for the whole project history?

File: www/scipylib/donations.rst [hunk 3]
```diff
@@ -57,6 +57,223 @@ infrastructure:
...
+------
+
+SciPy is a community project sponsored by Enthought, Inc.
+SciPy originated with code contributions by Travis Oliphant, Pearu
+Peterson, and Eric Jones.  Travis Oliphant and Eric Jones each contributed
+about half the initial code.  Pearu Peterson developed f2py, which is the
+integral to wrapping the many Fortran libraries used in SciPy.
+
+Below is a partial list of contributors to SciPy.  If you've
+been left off, please email the `SciPy Developers List`_.
```
Comment (rgommers) [inline www/scipylib/donations.rst:71]: Let's remove this as well, we don't want emails for this.

File: www/scipylib/donations.rst (hunk 1 above)
Comment (rlucas7) [reply www/scipylib/donations.rst:64]: Ok, I'm mostly just taking the material here directly from the repo. I'll remove this line.

File: www/scipylib/donations.rst (hunk 2 above)
Comment (rlucas7) [reply www/scipylib/donations.rst:75]: > How about we merge the `authors.py` feature first, and then we just make this a complete list for the whole project history?

Sounds good to me. For the time being I'll leave the file in this PR "as is"

File: www/scipylib/donations.rst (hunk 3 above)
Comment (rlucas7) [reply www/scipylib/donations.rst:71]: ok

**Comment by rgommers** [comment]
What I'd like to do with acknowledgements for SciPy:

- Once NEP 46 (https://github.com/numpy/numpy/pull/18084) is approved, I want to adapt it to SciPy.
- Use the all-contributors bot (see https://allcontributors.org/) for acknowledgements of every single contributor. It's then easy to do it both per release, and if we want to per individual if we feel that's warranted (in particular useful for non-code contributions, who will otherwise get missed by the auto-generated list).

So I'd like to keep this PR open as a reminder, but not merge it.

**Comment by rlucas7** [comment]
> What I'd like to do with acknowledgements for SciPy:
> 
>     * Once NEP 46 ([numpy/numpy#18084](https://github.com/numpy/numpy/pull/18084)) is approved, I want to adapt it to SciPy.
> 
>     * Use the all-contributors bot (see https://allcontributors.org/) for acknowledgements of every single contributor. It's then easy to do it both per release, and if we want to per individual if we feel that's warranted (in particular useful for non-code contributions, who will otherwise get missed by the auto-generated list).
> 
> 
> So I'd like to keep this PR open as a reminder, but not merge it.

Thanks for the comment here Ralf and good to know there is a bot for handling these sorts of things. 
Let me know if you need anything from me w.r.t. the all-contributors bot. I haven't used it before but can look into setting up/enabling for the repo unless someone else has already done so. I'm assuming the all-contributors bot intergrates w/github somehow.

**Comment by rgommers** [comment]
> I'm assuming the all-contributors bot intergrates w/github somehow.

It does. The trick is to figure out on which repo to enable it, and whether to use the default of adding people to some `README.md` or if it's easy to change to taking that generated html and putting it into a website.

**Comment by stefanv** [comment]
Shall we drop the THANKS.txt entirely?  It is not being kept up to date, and the git repository / GitHub has all the code contributor information.

**Comment by rlucas7** [comment]
+1 from me FWIW, contributions are in the log and in release notes.

-L

**Comment by rgommers** [comment]
That's four +1's, I'll add mine - should be enough to call it good. 

Thanks for this PR anyway Lucas!

**Pull request closed without merging**